recipes/tmpdir/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "@nodejs/tmpdir",
+  "version": "0.0.1",
+  "description": "Handle DEP0022 via transforming `tmpDir` to `tmpdir`.",
+  "type": "module",
+  "scripts": {
+    "test": "npx codemod@next jssg test -l javascript ./src/workflow.ts ./"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nodejs/userland-migrations.git",
+    "directory": "recipes/tmpdir",
+    "bugs": "https://github.com/nodejs/userland-migrations/issues"
+  },
+  "author": "nekojanai (Jana)",
+  "license": "MIT",
+  "homepage": "https://github.com/nodejs/userland-migrations/blob/main/recipes/tmpdir/README.md",
+  "devDependencies": {
+    "@types/node": "^24.0.3"
+  },
+  "dependencies": {
+    "@ast-grep/napi": "^0.38.6",
```

```suggestion
    "@ast-grep/napi": "^0.39.1",
```

It's should fix you type checking issue